packages/discord.js/src/structures/GuildMember.js

```diff
@@ -346,8 +346,8 @@ class GuildMember extends Base {
    * @param {BanOptions} [options] Options for the ban
    * @returns {Promise<GuildMember>}
    * @example
-   * // ban a guild member
-   * guildMember.ban({ deleteMessageDays: 7, reason: 'They deserved it' })
+   * // ban a guild member, deleting a week of messages
+   * guildMember.ban({ deleteMessageSeconds: 604800, reason: 'They deserved it' })
```

Comment:
Just like the [timeout](https://discord.js.org/#/docs/discord.js/main/class/GuildMember?scrollTo=timeout) example:

```suggestion
   * // Ban a guild member, deleting a week's worth of messages
   * guildMember.ban({ deleteMessageSeconds: 60 * 60 * 24 * 7, reason: 'They deserved it' })
```